pkg/vtpm/Dockerfile

```diff
@@ -23,9 +23,12 @@ RUN strip --strip-unneeded /out/usr/lib/libtpms.so.*
 
 # build swtpm
 WORKDIR /swtpm
-ADD https://github.com/stefanberger/swtpm.git#v0.9.0 /swtpm
+# This poinst to the latest commit that implements state backup functionality.
```

Comment:
Typo "poinst" - was that also in the master commit? If so we leave it here as well.

Reply:
Yes, it is in master.